public/manifest.json

```diff
@@ -28,12 +28,16 @@
     "default_title": "ROSE Wallet",
     "default_popup": "../extension/src/popup.html"
   },
-  "permissions": ["storage", "notifications", "activeTab"],
-  "content_security_policy": "{{{ EXTENSION_CSP }}}",
+  "permissions": ["storage", "notifications"],
+  "content_security_policy": {
+    "extension_pages": "{{{ EXTENSION_CSP }}}"
+  },
   "background": {
-    "page": "../extension/src/background.html",
-    "persistent": true
+    "service_worker": "service-worker.js"
   },
-  "web_accessible_resources": ["./oasis-xu-frame.html"],
-  "externally_connectable": { "ids": [] }
+  "externally_connectable": { "ids": [] },
```

Comment:
If we remove this entry error/warning in `chrome://extensions/` will be removed and the end result will be the same (no extensions or apps can connect).